### ISSUE TITLE
Bots wont show up inside Raid

### ISSUE THREAD

**Issue by NetherstormX** [trickerer/Trinity-Bots#233]
When i enter an instance/raid bot wont show inside and also bots group frame goes Unknown.

**Comment by trickerer** [comment]
https://github.com/trickerer/Trinity-Bots#raid-group-frames

**Comment by NetherstormX** [comment]
Sorry @trickerer , i guess i wasn't clear with my issue.
Let me explain again in detail:
I spawned a new bot got in group with him, (converted to raid) went to a raid instance, went inside the instance but the bot wont show up inside,
he wont spawn inside and the Frame goes unknown. I guess the solution that you provided is for fixing the unknown frame but how do i make him spawn inside the instance. 
(Instances that i tried are Zul'Aman and Naxxramas.)

Besides that i tried to go to a non-raid dungeon (i tried Hellfire rampards) and bot spawns inside when i get in. but not in raids.

More info:
I compiled the pre-patched repository that is provided and used the merge all required SQL files script for DB.
